src/scheduleMessage.ts

```diff
@@ -0,0 +1,23 @@
+import { Methods } from "@slack/web-api";
+import { DateTime } from "luxon";
+
+export async function scheduleLunchMessage(chat: Methods["chat"]) {
+  const response = await chat.scheduledMessages.list();
+
+  if (response.scheduled_messages?.length !== 0) {
+    return;
+  }
+
+  const date = DateTime
+    .now()
+    .setZone("Europe/Paris")
+    .startOf("day")
+    .plus({ days: 1, hours: 11, minutes: 30 })
```

Comment:
It will be nice if the value of `hours` and `minutes` come from an argument of the function `scheduleLunchMessage`.

Reply:
Ok, will check as well ;)

Reply:
054c96a9fec33a14cdb43b6c85dd189f37068ddb